### ISSUE TITLE
Thumbnail delay

### ISSUE THREAD

**Issue by BParks21** [libretro/RetroArch#14031]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description
In recent Nightly builds thumbnails are loading slower when browsing a collection of playlists. In older builds they load nearly instantly. 

### Expected behavior
Thumbnails should be loaded at the speed they were originally loading at. 

### Actual behavior
I was running RA Git 6f0bef6 and thumbnails were loading quickly as usual then updated to Git 13732ba and noticed they now have a slight delay. All settings are exactly the same across both versions of RA. I have the same playlists, thumbnails and games. Everything is also on the same drive. 

### Steps to reproduce the bug

Compare RA stable 1.10.3 to the latest nightly when navigating your playlists. Thumbnails will load slower as you navigate. See video comparisons. 
### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

Not sure when it started, possibly sooner than the version I updated to. 

### Environment information

- OS: Windows 10 Pro 64bit latest version
- Compiler: [In case you are running local builds]
[Videos.zip](https://github.com/libretro/RetroArch/files/8883064/Videos.zip)

**Comment by Cthulhu-throwaway** [comment]
Working as intended. See: https://github.com/libretro/RetroArch/pull/13947

**Comment by BParks21** [comment]
So to improve overall performance we get this delay, dang alright then. Well at least I know it is the intended behavior.

**Comment by PoloniumRain** [comment]
> Working as intended. See: #13947

I was about to open a new issue because of this, it's really bothering me. Can an option please be added to disable the thumbnail delay? It shouldn't be forced on everyone. The experience is now slower/worse for the vast majority of RA users who i would guess are not going to be using such extremely low-end hardware such as the Sony Vita where the fast thumbnail loading was causing problems.

**Comment by BParks21** [comment]
Yeah I agree I feel like user's with really high end hardware should be able to toggle the delay or something. The previous behavior was just a lot smoother.

**Comment by Cthulhu-throwaway** [comment]
I am not the author of that PR, you should tag him if you want more details.

**Comment by Augusto7743** [comment]
Thumbnail delay is an good feature when the user browse several file list thus avoiding unnecessary disk access.
An delay option allow adjust an value.

**Comment by i30817** [comment]
This might be a 'unwelcome' idea, but the delay only applies to downloads. So if you download the thumbnails you need ahead of time, you can avoid it.

Retroarch doesn't have the functionality to do this, but my python application, [libretro-fuzz](https://github.com/i30817/libretrofuzz) actually does that (and adapts the names of the the thumbnails to the names of the roms you actually have).

**Comment by BParks21** [comment]
What do you mean only applies to downloads? I have every thumbnail for every game i own already downloaded. There is always a delay even when everything has been downloaded.

**Comment by i30817** [comment]
I really didn't notice that. I do notice that if i keep the down button pressed in a playlist, the image will not appear until it's not pressed, but if i tap the button the image appears in less than 1/10 of a second (afaict, i didn't time it).

**Comment by i30817** [comment]
Yeah, that tracks with what i see here. A knob for this wouldn't be that weird although imo, I find it confusing that people can actually recognize thumbnails that appear for a frame or two, a way to show them when pressing down wouldn't be too bad.

Problem is, 'pressing down' accelerates now, not just speeds down at a fixed speed. People complained on large playlists.

**Comment by Augusto7743** [comment]
@i30817
Your Fuzzy Retroarch thumbnail downloader use playlist compressed ? Have an GUI version ?

**Comment by i30817** [comment]
No, i don't use the compressed playlists so i never noticed the lack. Largest i have is 800kb so i never bothered.

**Comment by i30817** [comment]
Ok, i think it supports compressed playlists now, or at least it managed to uncompress the few tests i made and not trip the assert of the total size mismatch.

**Comment by Augusto7743** [comment]
I will test if support compressed playlist.

**Comment by BParks21** [comment]
So now the thumbnails are loading quicker again as I scroll through my playlists on the latest stable build. However if I launch a game and start scrolling through other playlists certain playlists become slow to load thumbnails all of a sudden. I also notice that scrolling my games gets kind of jittery when this happens.
